query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -3,12 +3,20 @@ package filodb.query.exec
 import monix.reactive.Observable
 import org.jctools.queues.SpscUnboundedArrayQueue
 
-import filodb.core.query.{IteratorBackedRangeVector, RangeVector, ResultSchema}
+import filodb.core.metadata.Column.ColumnType
+import filodb.core.metadata.Dataset
+import filodb.core.query.{ColumnInfo, IteratorBackedRangeVector, RangeVector, ResultSchema}
 import filodb.memory.format.RowReader
 import filodb.query.{Query, QueryConfig, RangeFunctionId}
 import filodb.query.exec.rangefn.{RangeFunction, Window}
 import filodb.query.util.IndexedArrayQueue
 
+object PeriodicSamplesMapper {
+  // A PeriodicSamplesMapper always outputs timestamp:Long/value:Double right now
+  val periodicSchema = ResultSchema(Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
+                                        ColumnInfo("value", ColumnType.DoubleColumn)), 0)
```

Comment:
@velvia @tjackpaul  Number of row-key columns should be 1.
This is probably the reason for regression.